docs/api-routes/dynamic-api-routes.md

```diff
@@ -16,7 +16,7 @@ API routes support [dynamic routes](/docs/routing/dynamic-routes.md), and follow
 For example, the API route `pages/api/post/[pid].js` has the following code:
 
 ```js
-export default (req, res) => {
+export default function PostId(req, res){
```

Comment:
Let's either use `handler` for all the names, or something that focuses on what the function returns, e.g `getPost` (and doesn't start with an uppercase)

Reply:
Sounds good. I'll go ahead and fix that. Thanks!